.github/workflows/test-pr.yml

```diff
@@ -25,7 +24,23 @@ jobs:
   test-pr:
     name: 'Build and test Blockchain Plugin'
     runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: read
     steps:
+      - uses: actions/create-github-app-token@v1
+        id: temp-token
+        name: Generate temporary GitHub Token
+        with:
+          app-id: ${{ vars.CICD_GITHUB_WEBAPP_ID }}
+          private-key: ${{ secrets.CICD_GITHUB_WEBAPP_PRIVATE_KEY }}
+          owner: ${{ github.repository_owner }}
```

Comment:
I don't see where a build-output from this step is being used.

If I understand correctly, a build output from a prior step is referenced by doing:

```
${{ steps.step_name.var_name }}
```

but I can't find an output like that here. Perhaps this section can be removed?

Reply:
Removed! Thanks!